ee_tests/src/page_objects/user_profile.page.ts

```diff
@@ -59,26 +60,63 @@ export class CleanupUserEnvPage extends AppPage {
 
   async cleanup(username: string) {
     await this.eraseEnvButton.clickWhenReady();
-
     let confirmationElement =  this.innerElement(ui.BaseElement, 'modal', '');
     let confirmationBox = new CleanupConfirmationModal(confirmationElement);
 
     await confirmationBox.ready();
+
     await confirmationBox.confirmationInput.enterText(username);
     await confirmationBox.confirmEraseButton.clickWhenReady();
-    await browser.wait(until.presenceOf(
+
+    await browser.wait(until.presenceOf(this.alertBox), support.LONG_WAIT);
+    let alertText = await this.alertBox.getText();
+    support.info('Alert text = ' + alertText);
+
+    /* An intermittent error is resulting in some user enviroment resets failing
+       https://github.com/openshiftio/openshift.io/issues/1637
+       If this happens - try one more time - if that fails something serious
+       has happened and the test will/and should fail */
+
+    if (alertText.includes('Your OpenShift.io environment has been erased')) {
+      await browser.wait(until.presenceOf(
+      element(by.cssContainingText('fabric8-cleanup', 'Your OpenShift.io environment has been erased!'))));
```

Comment:
1. This string is there three times - refactor to a separate variable/constant.
2. You don't need to wait for text to appear because it's already there, just log the message